src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -426,7 +425,7 @@ object ExportVCF {
             if (tgs.isElementDefined(m, gsOffset, i))
               emitGenotype(sb, formatFieldOrder, tg, m, tgs.loadElement(m, gsOffset, localNSamples, i))
             else
-              sb.append("./.")
+              sb.append(missingFormatFieldString)
```

Comment:
This is bad because it makes the text much larger.  Omitted fields in the FORMAT field are assumed to be missing.  You want `.` instead of `./.` for genotypes?